specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2020-08-13-aml-preview/examples/Skus_List.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
All of the resources here are type caches. OTOH, we don't know our SKUs yet, so maybe there's nothing to do here?

Reply:
I can update this when we have skus for amlFilesystems.

Reply:
works for me